.travis.yml

```diff
@@ -1,9 +1,21 @@
 language: ruby
 rvm:
   - 2.3.0
-  - 2.2.3
```

Comment:
I suggest not removing old Ruby versions to ensure fudge stays compatible with projects depending on those Ruby versions.